packages/react/package.json

```diff
@@ -38,14 +38,14 @@
     "snapshot": "build-storybook && percy-storybook --widths=320,1280"
   },
   "peerDependencies": {
-    "carbon-components": "^10.9.0",
+    "carbon-components": "^10.30.0",
```

Comment:
oh dang, that was a big jump. Had we really not updated that since 10.9? 👀 crazy!

Reply:
😅